diff -urN gramps-1.0.7.orig/src/gramps.desktop gramps-1.0.7/src/gramps.desktop
--- gramps-1.0.7.orig/src/gramps.desktop	2004-12-03 15:42:01.416465976 +0100
+++ gramps-1.0.7/src/gramps.desktop	2004-12-03 15:42:30.052112696 +0100
@@ -1,23 +1,24 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=GRAMPS Genealogy System
-Name[sv]=GRAMPS
-Name[no]=GRAMPS
-Name[nb]=GRAMPS
-Name[ru]=GRAMPS
 Name[de]=GRAMPS
 Name[hu]=GRAMPS
+Name[nb]=GRAMPS
+Name[pl]=GRAMPS
+Name[sv]=GRAMPS
+Name[ru]=GRAMPS
+Name[sv]=GRAMPS
 Comment=Manage genealogical information, perform genealogical research and analysis
-Comment[sv]=Ett släktforskningsprogram
-Comment[no]=Et slektsforskningsprogram
-Comment[nb]=Et slektsforskningsprogram
-Comment[ru]=Система исследования и анализа генеалогической информации
 Comment[de]=Ein Programm zur Ahnenforschung
 Comment[hu]=Örökléskutató és elemző program
+Comment[nb]=Et slektsforskningsprogram
+Comment[pl]=Zarządzanie informacjami genealogicznymi, przeprowadzanie badań i analiz genealogicznych
+Comment[ru]=Система исследования и анализа генеалогической информации
+Comment[sv]=Ett släktforskningsprogram
 Exec=gramps
 Icon=gramps.png
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=Application;Genealogy;
+Categories=GTK;GNOME;Education;Science;
 X-GNOME-DocPath=gramps/gramps-manual.xml
