diff -urN gramps-2.2.3.orig/data/Makefile.am gramps-2.2.3/data/Makefile.am
--- gramps-2.2.3.orig/data/Makefile.am	2006-11-10 07:10:42.000000000 +0100
+++ gramps-2.2.3/data/Makefile.am	2006-12-11 23:36:12.649249750 +0100
@@ -34,7 +34,7 @@
 xmldir = $(SHARED_MIME_DIR)/packages
 xml_DATA = $(xml_files)
 
-pngdir = $(datadir)/icons/gnome/48x48/mimetypes
+pngdir = $(datadir)/icons/hicolor/48x48/mimetypes
 png_DATA = \
 	gnome-mime-application-x-gramps.png \
 	gnome-mime-application-x-gedcom.png \
@@ -42,7 +42,7 @@
 	gnome-mime-application-x-gramps-xml.png \
 	gnome-mime-application-x-geneweb.png
 
-svgdir = $(datadir)/icons/gnome/scalable/mimetypes
+svgdir = $(datadir)/icons/hicolor/scalable/mimetypes
 svg_DATA = \
 	gnome-mime-application-x-gramps.svg \
 	gnome-mime-application-x-gedcom.svg \
