diff -urN gramps-2.0.9.org/configure gramps-2.0.9/configure
--- gramps-2.0.9.org/configure	2005-12-12 02:41:21.000000000 +0100
+++ gramps-2.0.9/configure	2006-02-18 17:28:09.521722000 +0100
@@ -2023,7 +2023,7 @@
    { (exit 1); exit 1; }; }
 fi
 
-LANGUAGES="hu zh_CN cs da de es fr it nb nl no pl pt_BR ro ru sv eo fi lt"
+LANGUAGES="hu zh_CN cs da de es fr it nb nl pl pt_BR ro ru sv eo fi lt"
 
 
 DISTLANGS=
diff -urN gramps-2.0.9.org/configure.in gramps-2.0.9/configure.in
--- gramps-2.0.9.org/configure.in	2005-12-12 02:40:17.000000000 +0100
+++ gramps-2.0.9/configure.in	2006-02-18 17:28:39.101722000 +0100
@@ -42,7 +42,7 @@
   AC_MSG_ERROR(Couldn't find iconv program.)
 fi
 
-LANGUAGES="hu zh_CN cs da de es fr it nb nl no pl pt_BR ro ru sv eo fi lt"
+LANGUAGES="hu zh_CN cs da de es fr it nb nl pl pt_BR ro ru sv eo fi lt"
 AC_SUBST(LANGUAGES)
 
 DISTLANGS=
