diff -dur -x '*.orig' gramps-4.2.3.orig/setup.py gramps-4.2.3/setup.py
--- gramps-4.2.3.orig/setup.py	2016-04-10 17:59:24.000000000 +0200
+++ gramps-4.2.3/setup.py	2016-04-16 17:57:07.219083736 +0200
@@ -407,7 +407,7 @@
 #
 #-------------------------------------------------------------------------
 data_files_core = [('share/mime-info', ['data/gramps.mime']),
-                   ('share/icons', ['images/gramps.png'])]
+                   ('share/pixmaps', ['images/gramps.png'])]
 DOC_FILES = ['AUTHORS', 'COPYING', 'FAQ', 'INSTALL', 'LICENSE', 'NEWS',
              'README', 'TODO']
 GEDCOM_FILES = glob.glob(os.path.join('example', 'gedcom', '*.*'))
@@ -429,8 +429,8 @@
 
 PNG_FILES = glob.glob(os.path.join('data', '*.png'))
 SVG_FILES = glob.glob(os.path.join('data', '*.svg'))
-data_files_core.append(('share/icons/gnome/48x48/mimetypes', PNG_FILES))
-data_files_core.append(('share/icons/gnome/scalable/mimetypes', SVG_FILES))
+data_files_core.append(('share/icons/hicolor/48x48/mimetypes', PNG_FILES))
+data_files_core.append(('share/icons/hicolor/scalable/mimetypes', SVG_FILES))
 
 XML_FILES = glob.glob(os.path.join('data', '*.xml'))
 data_files_core.append(('share/gramps', XML_FILES))
