diff -urN gramps-2.2.3.orig/data/gramps.desktop gramps-2.2.3/data/gramps.desktop
--- gramps-2.2.3.orig/data/gramps.desktop	2006-11-27 00:13:20.000000000 +0100
+++ gramps-2.2.3/data/gramps.desktop	2006-12-12 00:35:12.031772000 +0100
@@ -32,7 +32,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=Application;Genealogy;Office;
+Categories=GTK;Gnome;Education;
 X-GNOME-DocPath=gramps/gramps-manual.xml
 MimeType=application/x-gramps
 Exec=gramps %F
