diff -urN gramps-2.0.11.orig/configure gramps-2.0.11/configure
--- gramps-2.0.11.orig/configure	2006-08-10 22:05:40.179292250 +0200
+++ gramps-2.0.11/configure	2006-08-10 22:07:48.995342750 +0200
@@ -2023,7 +2023,7 @@
    { (exit 1); exit 1; }; }
 fi
 
-LANGUAGES="hu zh_CN cs da de es fr it nb nl no pl pt_BR ro ru sv eo fi lt sk"
+LANGUAGES="hu zh_CN cs da de es fr it nb nl pl pt_BR ro ru sv eo fi lt sk"
 
 
 DISTLANGS=
diff -urN gramps-2.0.11.orig/configure.in gramps-2.0.11/configure.in
--- gramps-2.0.11.orig/configure.in	2006-08-10 22:05:40.179292250 +0200
+++ gramps-2.0.11/configure.in	2006-08-10 22:08:19.049221000 +0200
@@ -42,7 +42,7 @@
   AC_MSG_ERROR(Couldn't find iconv program.)
 fi
 
-LANGUAGES="hu zh_CN cs da de es fr it nb nl no pl pt_BR ro ru sv eo fi lt sk"
+LANGUAGES="hu zh_CN cs da de es fr it nb nl pl pt_BR ro ru sv eo fi lt sk"
 AC_SUBST(LANGUAGES)
 
 DISTLANGS=
