diff -urN gramps-2.0.8.org/src/data/gramps.desktop gramps-2.0.8/src/data/gramps.desktop
--- gramps-2.0.8.org/src/data/gramps.desktop	2005-10-22 22:17:30.358171000 +0200
+++ gramps-2.0.8/src/data/gramps.desktop	2005-10-23 12:57:08.399314000 +0200
@@ -1,28 +1,29 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=GRAMPS Genealogy System
-Name[sv]=GRAMPS
-Name[no]=GRAMPS
-Name[nb]=GRAMPS
-Name[ru]=GRAMPS
 Name[de]=GRAMPS
+Name[fi]=GRAMPS sukututkimusohjelma
 Name[hu]=GRAMPS
+Name[nb]=GRAMPS
 Name[nl]=GRAMPS genealogie
-Name[fi]=GRAMPS sukututkimusohjelma
+Name[pl]=GRAMPS
+Name[ru]=GRAMPS
+Name[sv]=GRAMPS
 Comment=Manage genealogical information, perform genealogical research and analysis
-Comment[nl]=Een programma voor genealogie infomatie beheer en genealogisch onderzoek en analyse
-Comment[sv]=Ett släktforskningsprogram
-Comment[no]=Et slektsforskningsprogram
-Comment[nb]=Et slektsforskningsprogram
-Comment[ru]=Система исследования и анализа генеалогической информации
 Comment[de]=Ein Programm zur Ahnenforschung
-Comment[hu]=Örökléskutató és elemző program
 Comment[fi]=Hallinnoi, tutki ja analysoi sukutietoa
+Comment[hu]=Örökléskutató és elemző program
+Comment[nb]=Et slektsforskningsprogram
+Comment[nl]=Een programma voor genealogie infomatie beheer en genealogisch onderzoek en analyse
+Comment[pl]=Zarządzanie informacjami genealogicznymi, przeprowadzanie badań i analiz genealogicznych
+Comment[ru]=Система исследования и анализа генеалогической информации
+Comment[sv]=Ett släktforskningsprogram
 Icon=gramps.png
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=Application;Genealogy;
+Categories=GTK;GNOME;Education;Science;
 X-GNOME-DocPath=gramps/gramps-manual.xml
 MimeType=application/x-gramps
 Exec=gramps %F
+# vi: encoding=utf-8
