--- gramps-4.0.2/setup.py~	2013-11-08 10:26:40.000000000 +0100
+++ gramps-4.0.2/setup.py	2013-12-04 17:18:23.232488702 +0100
@@ -415,8 +415,8 @@
 
 PNG_FILES = glob.glob(os.path.join('data', '*.png'))
 SVG_FILES = glob.glob(os.path.join('data', '*.svg'))
-data_files_core.append(('share/icons/gnome/48x48/mimetypes', PNG_FILES))
-data_files_core.append(('share/icons/gnome/scalable/mimetypes', SVG_FILES))
+data_files_core.append(('share/icons/hicolor/48x48/mimetypes', PNG_FILES))
+data_files_core.append(('share/icons/hicolor/scalable/mimetypes', SVG_FILES))
 
 XML_FILES = glob.glob(os.path.join('data', '*.xml'))
 data_files_core.append(('share/gramps', XML_FILES))
